packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1405,6 +1405,18 @@ const schema: SchemaType<DbUser> = {
     ...schemaDefaultValue(false)
   },
 
+  optedInToDialogueFacilitation: {
+    type: Boolean,
+    canRead: [userOwns, 'sunshineRegiment', 'admins'],
+    canCreate: ['members'],
+    canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
+    optional: true,
+    nullable: false,
+    hidden: forumTypeSetting.get() !== 'LessWrong',
```

Comment:
This should use `!isLW` instead (and if previous PRs didn't, that was my oversight)